common/services/prismic/api.js

```diff
@@ -56,9 +56,9 @@ export async function getDocuments(
       ? memoizedPrismic
       : await getPrismicApi(req);
   const docs: PrismicApiSearchResponse = await prismicApi.query(
-    predicates.concat([Prismic.Predicates.not('document.tags', ['delist'])]),
+    // predicates.concat([Prismic.Predicates.not('document.tags', ['delist'])]),
```

Comment:
This need to come back in

Reply:
oh darn it I've been pushing changes to the wrong branch. Sorry, I'll clear this up.

Reply:
Tidied it up so it's just the stuff I wanted reviewed now. Sorry about that.